lib/postgresql.js

```diff
@@ -15,6 +15,11 @@ var ParameterizedSQL = SqlConnector.ParameterizedSQL;
 var util = require('util');
 var debug = require('debug')('loopback:connector:postgresql');
 
+// Add support for node 0.10.x
+if (typeof Promise == 'undefined') {
+  global.Promise = require('promise-polyfill');
```

Comment:
> We usually use bluebird instead of promise-polyfill. I am proposing to use bluebird here too, for the sake of consistency.

I tried changing it to `promise` earlier but CI still failed.